py-polars/polars/datatypes.py

```diff
@@ -44,7 +36,7 @@
     "DTYPE_TO_FFINAME",
     "date_like_to_physical",
     "dtype_to_ctype",
-    "pytype_to_polars_type",
```

Comment:
There were two different functions that did more or less the same, hence you will see this rename in a number of places.